ide/css.lib/src/org/netbeans/modules/css/lib/api/properties/Properties.java

```diff
@@ -144,24 +146,26 @@ public static boolean isAggregatedProperty(FileObject context, PropertyDefinitio
         final GroupGrammarElement grammarElement = propertyDefinition.getGrammarElement(context);
         final AtomicBoolean isAggregated = new AtomicBoolean(false);
         grammarElement.accept(new GrammarElementVisitor() {
-            private boolean cancelled = false;
+            private final Map<GroupGrammarElement,Void> seen = new IdentityHashMap<>();
```

Comment:
It's ok. Maybe`Collections.newSetFromMap(new IdentityHashMap<>()) `  makes things clearer below (`seen.contains`, `seen.add`, etc.)?

Reply:
Addressed via https://github.com/apache/netbeans/pull/5432/commits/e2dd93481e809985df55c138448cbaaf0ab75f4b
Learned something new here - thanks.